tests/personal_data/portfolios/test_asset.py

```diff
@@ -52,3 +52,11 @@ def test__ensure_update_validity():
         asset._ensure_update_validity(constants.ONE, -constants.ONE)
         with pytest.raises(errors.PortfolioNegativeValueError):
             asset._ensure_update_validity(-constants.ONE, constants.ZERO)
+
+def test__ensure_not_negative():
+    asset = assets.Asset(ASSET_CURRENCY_NAME, available=constants.ONE_HUNDRED, total=decimal.Decimal(2))
+    if not os.getenv('CYTHON_IGNORE'):
```

Comment:
`_ensure_not_negative` is cpdef, it should work in cython ?

Reply:
right :+1: